<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiMenu</class>
 <widget class="QMainWindow" name="GuiMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1462</width>
    <height>887</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>GuiMenu</string>
  </property>
  <widget class="QWidget" name="Widget">
   <widget class="QWidget" name="TestResult" native="true">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>23</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <widget class="QPushButton" name="TestResultView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Result Info</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="TestResultProgressBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <zorder>TestResultProgressBar</zorder>
    <zorder>TestResultView</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1462</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionLastFolder"/>
    <addaction name="actionLastFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQT"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionInsert"/>
    <addaction name="actionDelete"/>
    <addaction name="actionMergeSplit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="actionNodeView"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionAppMode"/>
    <addaction name="actionSFCModelPath"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionGenTC"/>
    <addaction name="actionRunTC"/>
    <addaction name="actionRunTCMulti"/>
    <addaction name="actionTCReport"/>
    <addaction name="actionGCOVReport"/>
    <addaction name="separator"/>
    <addaction name="actionEnterScriptText"/>
    <addaction name="actionGenSSFS"/>
    <addaction name="separator"/>
    <addaction name="actionViewScriptLog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSetting"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionMergeSplit"/>
   <addaction name="separator"/>
   <addaction name="actionNodeView"/>
   <addaction name="separator"/>
   <addaction name="actionLastFolder"/>
   <addaction name="actionLastFile"/>
   <addaction name="separator"/>
   <addaction name="actionPath"/>
   <addaction name="actionRedo"/>
   <addaction name="actionUndo"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../Resource/Image.qrc">
     <normaloff>:/Image/Menu/Open.png</normaloff>:/Image/Menu/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../Resource/Image.qrc">
     <normaloff>:/Image/Menu/New.png</normaloff>:/Image/Menu/New.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../Resource/Image.qrc">
     <normaloff>:/Image/Menu/Save.png</normaloff>:/Image/Menu/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../Resource/Image.qrc">
     <normaloff>:/Image/Menu/Copy.png</normaloff>:/Image/Menu/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../../Resource/Image.qrc">
     <normaloff>:/Image/Menu/Cut.png</normaloff>:/Image/Menu/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../Resource/Image.qrc">
     <normaloff>:/Image/Menu/Paste.png</normaloff>:/Image/Menu/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Config</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionNodeView">
   <property name="text">
    <string>Node View</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="actionAppMode">
   <property name="text">
    <string>App Mode</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionSFCModelPath">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SFC Model Path</string>
   </property>
   <property name="iconText">
    <string>Path :</string>
   </property>
   <property name="toolTip">
    <string>SFC Model Path</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGenTC">
   <property name="text">
    <string>Gen TC</string>
   </property>
   <property name="shortcut">
    <string>Alt+G</string>
   </property>
  </action>
  <action name="actionRunTC">
   <property name="text">
    <string>Run TC</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionTCReport">
   <property name="text">
    <string>TC Report</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actionGCOVReport">
   <property name="text">
    <string>GCOV Report</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionEnterScriptText">
   <property name="text">
    <string>Enter Script Text</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionViewScriptLog">
   <property name="text">
    <string>View Script Log</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionGenSSFS">
   <property name="text">
    <string>Gen SSFS</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQT">
   <property name="text">
    <string>AboutQT</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPath">
   <property name="text">
    <string>Path :</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Insert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionMergeSplit">
   <property name="icon">
    <iconset resource="../../Resource/Image.qrc">
     <normaloff>:/Image/Menu/MergeSplit.png</normaloff>:/Image/Menu/MergeSplit.png</iconset>
   </property>
   <property name="text">
    <string>Merge/Split</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionRunTCMulti">
   <property name="text">
    <string>Gen TC New</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionLastFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LastFile</string>
   </property>
  </action>
  <action name="actionLastFolder">
   <property name="text">
    <string>LastFolder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Resource/Image.qrc"/>
 </resources>
 <connections/>
</ui>
